scripts/run_e2e_tests.py

```diff
@@ -419,7 +419,7 @@ def start_google_app_engine_server(dev_mode_setting, log_level):
     app_yaml_filepath = 'app%s.yaml' % ('_dev' if dev_mode_setting else '')
 
     p = subprocess.Popen(
-        '%s %s/dev_appserver.py --host 0.0.0.0 --port %s '
+        'yes | %s %s/dev_appserver.py --host 0.0.0.0 --port %s '
```

Comment:
Add a comment to explain this "yes" thing.

Also why is it needed here? I thought it was just for setup.

Reply:
OK, so it looks like the installation of additional components is triggered only when dev_appserver is first started. Could you look into a way to trigger the installation in the setup stage only, so that you don't need to modify all the other commands to dev_appserver?

Reply:
I looked up the docs for this. You should be able to just do something like the following on-setup:

```
gcloud components install app-engine-python app-engine-python-extras cloud-datastore-emulator --quiet
```

If you look at the gcloud help manual, the --quiet command disables interactive prompts.

Reply:
Ah I'm so sorry I should have updated the PR earlier more specifically. I actually found the --quiet flag at the very beginning but I initially thought it would work for dev_appserver which it doesn't. I eventually ended up figuring out this exact installation but it took a while because I didn't find the exact libraries involved in the installation prompt to be offered in the components list so I tried to look for a cleaner solution. I perused the code of gcloud install.sh and gcloud init to see if they could give us what I want (it did not) => eventually found out that installing those 3 libraries actually installs everything we need. 

The reason I haven't pushed the code yet is because I'm trying to figure out why the lighthouse tests are failing so will push when that is done.